### ISSUE TITLE
Unclosed tag in xml: Line number reported by validate() should be the opening tag's line

### ISSUE THREAD

**Issue by Gei0r** [NaturalIntelligence/fast-xml-parser#370]
* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [x] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Description
When there's an unclosed xml tag in the input, the `validate()` method does not output the line of the unclosed tag. Instead, the line of the parent's closing tag is reported as the error location. This is understandable, because this is the location where it's certain that there is an error (other parsers do the same). But it's not very helpful in finding the input error.

If possible, the error message should be changed to point to the unclosed tag.

### Code
```js
const fastXmlParser = require('fast-xml-parser');

const xml =
`<parent>
  <child name="Peter"/>
  <child name="Paul">      <!-- ERROR: should be self-closing -->
  <child name="Mary"/>
</parent>`;

console.log(fastXmlParser.validate(xml));
```

### Output
```js
{
  err: {
    code: 'InvalidTag',
    msg: "Closing tag 'child' is expected inplace of 'parent'.",
    line: 5   // <<<< not helpful
  }
}
```
### Desired Output
```js
{
  err: {
    code: 'InvalidTag',
    msg: "Unclosed tag 'child'",
    line: 3       // <<<< line of the opening tag
  }
}
```

**Would you like to work on this issue?**

- [x] Yes
- [ ] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates.
